ui/job-view/context/PinnedJobs.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { withNotifications } from '../../shared/context/Notifications';
 
 const COUNT_ERROR = 'Max pinboard size of 500 reached.';
 const MAX_SIZE = 500;
 const PinnedJobsContext = React.createContext({});
 
-export class PinnedJobs extends React.Component {
+export class PinnedJobsClass extends React.Component {
```

Comment:
The unit tests don't currently work well with the Context ``HOC``s.  So if I ``export`` the ``Class`` version, but then ``default export`` the ``HOC`` version, this seems to work well.  I'm hopeful ``Jest`` will have a better option.  I did this for a couple components.